README.md

```diff
@@ -1 +1,28 @@
-# My-Portfolio
\ No newline at end of file
+# My-Portfolio
+
+
+> My portfolio page.
+
+
+## Built With
+
+- html
+- css
+
+
+
+## Authors
+
+👤 **George kinyanjui**
+
+- GitHub: [@george827](https://github.com/george827)
+
+
+## Acknowledgments
+- Inspiration
+- [Microverse](https://github.com/microverseinc)
+
+
+## 📝 License
+
+This project is [MIT](https://github.com/george827/My-Portfolio/blob/main/LICENSE) licensed.
```

Comment:
- [ Optional ] Good job for documenting your README file in professional way, To Make it better I recommend you to add  "Getting Started" header section in your README file.
![Capture](https://user-images.githubusercontent.com/93598189/192005508-80442786-7421-42fb-b707-d8628f507ec9.PNG)